package-lock.json

```diff
@@ -1,33 +1,7326 @@
 {
   "name": "vscode-apache-camel",
   "version": "0.4.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
would have been better to keep the same version but well, will be boring to redo the change and risk is low